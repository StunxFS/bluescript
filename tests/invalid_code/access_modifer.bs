mod my_mod {
    const x = 1;
    pub(pkg) const rs = 2014;
    pub const v = 4;

    fn my_fn() {
        const my_x = x; // ok
        const my_rs = rs; // ok
        const my_v = v; // ok
    }
}

fn main() {
    const my_x = my_mod::x; // error
    const my_rs = my_mod::rs; // ok
    const my_v = my_mod::v; // ok
}