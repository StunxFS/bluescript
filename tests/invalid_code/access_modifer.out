tests/invalid_code/access_modifer.bs:14:26: error: cannot access private constant `x`
